.editorconfig

```diff
@@ -0,0 +1,20 @@
+# EditorConfig is awesome: http://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+indent_style = tab
+end_of_line = lf
+insert_final_newline = true
+charset = utf-8
+
+# Matches the exact files either package.json or .travis.yml
```

Comment:
copypasta. Removed this part of the file as it doesn't even apply to this project.